doc/api/cli.md

```diff
@@ -694,6 +710,7 @@ Node.js options that are allowed are:
 - `--experimental-report`
 - `--experimental-vm-modules`
 - `--force-fips`
+- `--frozen-intrinsics`
```

Comment:
NODE_OPTIONS is supported by using AddOption right? Or is there more I need to do here?

Reply:
Nope, that’s all you need to do. :)